src/transformers/models/clap/modeling_clap.py

```diff
@@ -1956,8 +1955,8 @@ def __init__(self, config: ClapConfig):
         text_config = config.text_config
         audio_config = config.audio_config
 
-        self.logit_scale_a = nn.Parameter(torch.tensor(np.log(config.logit_scale_init_value)))
```

Comment:
The aforementioned behaviour is a result of the `np.log` operation defaulting to float64

Reply:
Give the [original code](https://github.com/LAION-AI/CLAP/blob/6b1b4b5b4b87f4e19d3836d2ae7d7272e1c69410/src/laion_clap/clap_module/model.py#L545) we might need to init in float64 then cast to `float` if it makes a difference. No idea if the actual value save is in float64!

Reply:
The parameters are initialised in float64 but are stored in float32 in the state dict